README.md

```diff
@@ -1,53 +1,73 @@
 ---
 page_type: sample
 languages:
-- csharp
+- javascript
 products:
-- dotnet
-description: "Add 150 character max description"
-urlFragment: "update-this-to-unique-url-stub"
+- msal
+description: "A simple JavaScript Single-Page Application using @azure/msal-browser"
+urlFragment: "README"
 ---
 
-# Official Microsoft Sample
+# MSAL 2.x JavaScript Sample Application
 
-<!-- 
-Guidelines on README format: https://review.docs.microsoft.com/help/onboard/admin/samples/concepts/readme-template?branch=master
-
-Guidance on onboarding samples to docs.microsoft.com/samples: https://review.docs.microsoft.com/help/onboard/admin/samples/process/onboarding?branch=master
-
-Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
--->
-
-Give a short description for your sample here. What does it do and why is it important?
+A simple vanilla JavaScript single-page application which demonstrates how to configure [MSAL.JS 2.x](https://www.npmjs.com/package/@azure/msal-browser) to login, logout, and acquire an access token for a protected resource such as Microsoft Graph API.
 
 ## Contents
 
-Outline the file contents of the repository. It helps users navigate the codebase, build configuration and any related assets.
-
 | File/folder       | Description                                |
 |-------------------|--------------------------------------------|
-| `src`             | Sample source code.                        |
+| `app`             | Contains sample source files               |
+| `authPopup.js`    | Main authentication logic resides here (using Popup flow).            |
+| `authRedirect.js` | Use this instead of `authPopup.js` for authentication with redirect flow.   |
+| `authConfig.js`   | Contains configuration parameters for the sample. |
+| `graph.js`        | Provides a helper function for calling MS Graph API.   |
+| `graphConfig.js`  | Contains API endpoints for MS Graph.       |
+| `ui.js`           | Contains UI logic.                         |
+| `index.html`      |  Contains the UI of the sample.            |
 | `.gitignore`      | Define what to ignore at commit time.      |
-| `CHANGELOG.md`    | List of changes to the sample.             |
+| `changelog.md`    | List of changes to the sample.             |

```

Comment:
the file name itself is in capitals though :/

Reply:
It shouldn't be?

Reply:
they should be in the same form i would say.. sorry, just being picky :-))